### PR TITLE
Draft PR: consistent use of emissivity and albedo between the radiation and the LSMs. Also, make the radiation_surface.f work for fractional grid.

### PR DESCRIPTION
**Pull request by tanyasmirnova**
This PR has:
1. Options 2 for albedo and emissivity (ialb and iemis) that will provide albedo and emissivity computed in the LSMs to the SW and LW radiation. These new options work with Noah MP and RUC LSMs. Old options 1 should be used for the Noah LSM.
2. Changes in radiation_surface.f that will work correctly with/without fractional grid.

### CODE REVIEW

File: physics/radlw_main.F90 [hunk 1]
```diff
@@ -8854,25 +8854,25 @@ subroutine cldprmc(nlayers, inflag, iceflag, liqflag, cldfmc,     &
                   abscosno(ig) = 0.0_rb
 
                elseif (iceflag .eq. 0) then
-!                  if (radice .lt. 10.0_rb) stop 'ICE RADIUS TOO SMALL'
-                  abscoice(ig) = absice0(1) + absice0(2)/max(radice,10.0_rb)
+                  if (radice .lt. 10.0_rb) stop 'ICE RADIUS TOO SMALL'
+                  abscoice(ig) = absice0(1) + absice0(2)/radice
```
Comment (DomHeinzeller) [inline physics/radlw_main.F90:8858]: This reverts the changes that Greg Thompson and I agreed on in order to make his updated scheme run (there are no limits to the effective radii in his updated scheme).

**Review by DomHeinzeller — CHANGES_REQUESTED** [review]
Need to update from gsl/develop after the tendency PR was merged yesterday. There are many changes in this PR that make me concerned, such as the reversal on effective radii limits in the radiation code that Greg Thompson and I added on purpose. Also, the logic is getting incredibly complicated and composite physics get mixed with radiation physics in places deep down in the code. I don't know if there is an easier way to handle this in the current physics code, but it makes me worried.

File: physics/radlw_main.F90 (hunk 1 above)
Comment (tanyasmirnova) [reply physics/radlw_main.F90:8858]: @DomHeinzeller Dom, i guess it is the results of starting form older code in Mike B. NCAR directory. Should I make a new gsl/develop clone and merge my changes into this updated code?

**Comment by DomHeinzeller** [comment]
Either way you do it is fine, but your suggestion is probably the cleanest way. Let's discuss first with Ben today, just in case this will lead to changes to your PR?

**Comment by DomHeinzeller** [comment]
This PR has been pulled into a PR for NCAR master, and updated there. See https://github.com/NCAR/ccpp-physics/pull/627.

**Pull request closed without merging**